ethereum/contracts/IncentivizedSendChannel.sol

```diff
@@ -0,0 +1,25 @@
+// SPDX-License-Identifier: MIT
+pragma solidity >=0.6.2;
+pragma experimental ABIEncoderV2;
+
+import "./SendChannel.sol";
+
+// IncentivizedSendChannel is a channel that sends ordered messages with an increasing nonce. It will have incentivization too.
+contract IncentivizedSendChannel is SendChannel {
+    uint256 public currentNonce;
```

Comment:
ditto. See my previous comment about how the `current` prefix can be problematic.